new-integration-tests/src/test/java/oracle/weblogic/kubernetes/logging/LoggingFacade.java

```diff
@@ -0,0 +1,629 @@
+   *
+   * @param msg the message to log
+   * @param thrown an Exception to include in the logged message
+   */
+  public void severe(String msg, Throwable thrown) {
+    if (isSevereEnabled()) {
+      CallerDetails details = inferCaller();
+      logger.logp(Level.SEVERE, details.clazz, details.method, msg, thrown);
+    }
+  }
+
+  /**
+   * Logs that an exception will be thrown. The calling class and method names will be inferred.
+   *
+   * @param pending an Exception to include in the logged message
+   */
+  public void throwing(Throwable pending) {
+    if (isFinerEnabled()) {
+      CallerDetails details = inferCaller();
+      logger.throwing(details.clazz, details.method, pending);
+    }
+  }
+
+  /**
+   * Logs a message at the WARNING level.
+   *
+   * @param msg the message to log
+   */
+  public void warning(String msg) {
+    if (isWarningEnabled()) {
+      CallerDetails details = inferCaller();
+      logger.logp(Level.WARNING, details.clazz, details.method, msg);
+    }
+  }
+
+  /**
+   * Logs a message which requires parameters at the WARNING level.
+   *
+   * @param msg the message to log
+   * @param params varargs list of objects to include in the log message
+   */
+  public void warning(String msg, Object... params) {
+    if (isWarningEnabled()) {
+      CallerDetails details = inferCaller();
+      logger.logp(Level.WARNING, details.clazz, details.method, msg, params);
+    }
+  }
+
+  /**
+   * Logs a message which accompanies a Throwable at the WARNING level.
+   *
+   * @param msg the message to log
+   * @param thrown an Exception to include in the logged message
+   */
+  public void warning(String msg, Throwable thrown) {
+    if (isWarningEnabled()) {
+      CallerDetails details = inferCaller();
+      logger.logp(Level.WARNING, details.clazz, details.method, msg, thrown);
+    }
+  }
+
+  /**
+   * Logs a trace message with the ID FMW-TRACE at the FINER level.
+   *
+   * @param msg the message to log
+   */
+  public void trace(String msg) {
+    finer(TRACE + msg);
+  }
+
+  /**
+   * Logs a trace message with the ID FMW-TRACE at the FINER level.
+   *
+   * @param msg the message to log
+   * @param args parameters to the trace message
+   */
+  public void trace(String msg, Object... args) {
+    finer(TRACE + msg, args);
+  }
+
+  /**
+   * Returns a formatted message.
+   *
+   * @param msg the message to be formatted, which is key to the resource bundle
+   * @param args parameters to the message
+   * @return A formatted message
+   */
+  public String getFormattedMessage(String msg, Object... args) {
+    try {
+      return MessageFormat.format(logger.getResourceBundle().getString(msg), args);
+    } catch (Exception ex) {
+      return msg;
+    }
+  }
+
+  /**
+   * Obtains caller details, class name and method, to be provided to the actual Logger. This code
+   * is adapted from ODLLogRecord, which should yield consistency in reporting using PlatformLogger
```

Comment:
this comment looks like it is incorrect

Reply:
You added it to the code.... You should know.